CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+Thanks for contributing to WordCamp.org! This page will explain all the ways you can contribute.
```

Comment:
`SHOULD` - I think this should live in `.github/` to avoid cluttering the root, and since we'll probably want to add issue templates, etc.